tools/run-tests.py

```diff
@@ -195,6 +197,45 @@ def run_jsc_stress(engine, arch):
         env={'PYTHONPATH': '.'})
 
 
+def _run_regression_tests(files, is_fail):
+    for file in files:
+        passed = True
+        msg = ''
+        code = 0
+        try:
+            out = subprocess.check_output([DEFAULT_ESCARGOT, file])
+        except subprocess.CalledProcessError as exc:
+            passed = False
+            code = exc.returncode
+            msg = exc.output
+        finally:
+            if passed != is_fail:
+                print('%sOK: %s%s' % (COLOR_GREEN, file, COLOR_RESET))
+            else:
+                print('%sFAIL(%d): %s failed %s\n' % (COLOR_RED, code, file, COLOR_RESET))
+                return msg
```

Comment:
Shouldn't return too early, before executing all test cases.

Reply:
That's something I've been thinking of as well:
 - Do we want to run every single regression test even if the first one fails? The positive side of this is that it shows every single failure. The downside is the whole test-suite is executed, even if the first test fails, and the Travis job would fail anyways.

However I do agree with you on this one, as for the developer it makes it a lot easier to identify all of the bugs. without  having to run the test-suite all over again.

Reply:
We definitely want to run all existing tests in all test suites. Seeing an overview/summary about the passing/failing tests gives lots of information. It makes a big difference wheter only one test cases fails or when all tests fail. The developer gets much more insight. If we bail out at the first problem, we lose that. (All other test suites work like that for the exact same reason.)